.github/workflows/docker-scan.yml

```diff
@@ -0,0 +1,32 @@
+name: "docker-scan"
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 1'
```

Comment:
Just a heads up, this appears to run at midnight every Monday - is that what you're looking to do?